mmv1/templates/terraform/constants/billing_budget.erb

```diff
@@ -0,0 +1,25 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2022 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+
+// Check to see if a specified value in the config exists and supress diffs if so. Otherwise run emptyOrDefaultStringSuppress.
```

Comment:
```suggestion
// Check to see if a specified value in the config exists and suppress diffs if so. Otherwise run emptyOrDefaultStringSuppress.
```
we have a spell check in our linter